lib/iris/tests/unit/analysis/test_PERCENTILE.py

```diff
@@ -147,23 +148,35 @@ def test_masked_2d_multi(self):
             data, axis, percent, expected, mdtol=mdtol, approx=True
         )
 
-    @mock.patch("scipy.stats.mstats.mquantiles")
+    @mock.patch("scipy.stats.mstats.mquantiles", return_value=[2, 4])
```

Comment:
I'm not sure that it makes sense for these two checks to be in the `MaskedAggregateMixin` because they don't use masked data? I think they belong in `Test_aggregate` instead.

Reply:
Yeah, maybe that's the most straightforward way? (or a `SlowMixin`, but we're already pretty far into `Mixin`s so that may well get confusing).

Reply:
Or just rename this Mixin?

Reply:
Yeah - call it the `Slow` or the `Scipy` or something then the docstring can explain what belongs in here (masked and scipy mocking)?

Reply:
done